build broken/CMake complains about missing C++14 support in clang on macOS

I'm trying to build on macOS 10.15.4. All deps are installed.
I could install LLVM (via homebrew) but I'd prefer if I can the installed one from Apple.

Anyone seen this – is there a straightforward solution?
```
❯ make
-- Building OSL 1.11.6.0
-- CMake version is 3.17.3
-- Configuring OSL 1.11.6.0
-- CMake 3.17.3
-- CMake system           = Darwin-19.5.0
-- CMake system name      = Darwin
-- Project source dir     = /Users/moritz/code/OpenShadingLanguage
-- Project build dir      = /Users/moritz/code/OpenShadingLanguage/build/macosx
-- Project install prefix = /Users/moritz/code/OpenShadingLanguage/dist/macosx
-- Configuration types    =
-- Build type             = Release
-- Setting Namespace to: OSL_v1_11
-- CMAKE_CXX_COMPILER     = /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- CMAKE_CXX_COMPILER_ID  = AppleClang
-- Building for C++11
-- clang-format found: /usr/local/bin/clang-format
--
-- * Checking for dependencies...
-- *   - Missing a dependency 'Package'?
-- *     Try cmake -DPackage_ROOT=path or set environment var Package_ROOT=path
-- *     For many dependencies, we supply src/build-scripts/build_Package.bash
-- *   - To exclude an optional dependency (even if found),
-- *     -DUSE_Package=OFF or set environment var USE_Package=OFF
--
-- Found Boost 107200
-- Found ZLIB 1.2.11
-- Found OpenEXR 2.5.1
-- Found OpenImageIO 2.1.16.0
-- Found PugiXML
-- Found LLVM 10.0.0
CMake Error at src/cmake/externalpackages.cmake:247 (message):
  LLVM 10+ requires C++14 or higher (was 11).
Call Stack (most recent call first):
  CMakeLists.txt:134 (include)
```

--- a/src/cmake/externalpackages.cmake
+++ b/src/cmake/externalpackages.cmake
@@ -244,7 +244,13 @@ if (LLVM_NAMESPACE)
 endif ()
 if (LLVM_VERSION VERSION_GREATER_EQUAL 10.0.0 AND
     CMAKE_CXX_STANDARD VERSION_LESS 14)
-    message (FATAL_ERROR "LLVM 10+ requires C++14 or higher (was ${CMAKE_CXX_STANDARD}).")
+    message (FATAL_ERROR
+             "LLVM 10+ requires C++14 or higher (was ${CMAKE_CXX_STANDARD}). "
+             "To build against this LLVM ${LLVM_VERSION}, you need to set "
+             "build option CMAKE_CXX_STANDARD=14. The minimum requirements "
+             "for that are gcc >= 5.1, clang >= 3.5, Apple clang >= 7, "
+             "icc >= 7, MSVS >= 2017. "
+             "If you must use C++11, you need to build against LLVM 9 or earlier.")
 endif ()
 
 checked_find_package (partio)

